linux_os/guide/system/software/gnome/dconf_use_text_backend_gdm_profile/rule.yml

```diff
@@ -0,0 +1,37 @@
+documentation_complete: true
+
+prodtype: rhel7,rhel8,fedora,ol7,ol8
+
+title: 'Force dconf to use the textfiles instead of a binary DB GDM'
+
+description: |-
+    By default, DConf uses a binary database as a data backend.
+    The database is compiled from config files by the <pre>dconf update</pre> command.
+
+    dconf can be configured to look into those text files directly by inserting the
+    <pre>service-db:keyfile/user</pre>
+    directive at the beginning of the <pre>/etc/dconf/profile/user</pre> and
+    <pre>/usr/share/dconf/profile/gdm</pre> files.
+
+rationale: |-
+    Unlike text config files, the binary database is impossible to check by OVAL.
```

Comment:
`Unlike text config files, the binary database is impossible to check by OVAL.` should be `Because a dconf OVAL probe does not exist yet, the binary database cannot be evaluated by OVAL.`